src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -4688,7 +4688,7 @@ class MyAttrAttribute : Attribute
 {
 }
 
-[MyAttr(123, [|Version|] = 1)]
+[MyAttr(123, [|Value|] = 1)]
```

Comment:
why?

Reply:
The old code produced an [error](https://sharplab.io/#v2:D4AQTAjAsAULDaBBALsgTgSwEYFdkFMBVAZwEMBzfAChXWz3wBVS1LliA6AYQBtTjiASgC6scAAIAsgE9aaOfQLiAXOIW4CsAN6wAvrAQy5VCGADMAGnEA1fGmIYA9gDtxAXnEQRYsOIAi2nqwQA).

`error CS1069: The type name 'Version' could not be found in the namespace 'System'. This type has been forwarded to assembly 'System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' Consider adding a reference to that assembly.`

I could try and figure out exactly what is wrong with assembly references and redirects, but I don't think this test is worth risking my sanity ;-)